File: src/main/kotlin/com/github/tobi812/sprykerplugin/models/resolver/PathResolver.java
package com.github.tobi812.sprykerplugin.models.resolver;

public class PathResolver implements PathResolverInterface {

    private String basePath;

    public PathResolver(String basePath) {
        this.basePath = basePath;
    }

    public String resolvePath(String namespace) {
        String relativePath = namespace.replace("\\", "/");

        return basePath + "/" + relativePath;
    }

}

File: src/main/kotlin/com/github/tobi812/sprykerplugin/models/resolver/PathResolver.kt
package com.github.tobi812.sprykerplugin.models.resolver

class PathResolver(private val basePath: String) : PathResolverInterface {
    override fun resolvePath(namespace: String): String {
        val relativePath = namespace.replace("\\", "/")
        return "$basePath/$relativePath"
    }
}